.github/workflows/ci-daily.yml

```
@@ -37,6 +37,9 @@ jobs:
     - name: Install package dependencies
       run: yarn install --frozen-lockfile --ignore-scripts --prefer-offline
 
+    - name: Check for file changes
```

```suggestion
    - name: Check for change files
```
A bit of a nit, but I think we should be specific here. This checks for the presence of "change" files for the purpose of ensuring we have metadata necessary to generate a changelog and version correctly. So, not technically "file changes" as the PR has that *always*, we really want to look for the specific files.

---

We could also include "Check for the presence of changed files inside `./change`".  The entire detail should be added to documentation we wrote today @Ibrahimmaga.